parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestParquetReader.java

```diff
@@ -46,10 +47,19 @@
 
   private static final Path FILE_V1 = createTempFile();
   private static final Path FILE_V2 = createTempFile();
-  private static final List<PhoneBookWriter.User> DATA = Collections.unmodifiableList(makeUsers(10000));
+  private static final Path STATIC_FILE_WITHOUT_COL_INDEXES = createPathFromCP("/test-file-with-no-column-indexes-1.parquet");
```

Comment:
I am not sure if it is a good idea to check in a data file. Can you check if it is possible to stop generating offset index in the current version of Parquet?

Reply:
@shangxinli It looks like the [column-indexes](https://issues.apache.org/jira/browse/PARQUET-1201) are always written in the current version of parquet and are not configurable.
We are already testing the new row index support with and without the column index filtering being triggered (as part of TestColumnIndexFiltering). Also the new row index feature doesn't rely on column indexes in any way. So we can skip the backward compatibility testing and remove this parquet file from resources. What do you think about this?